source/MaterialXRenderGlsl/TextureBaker.cpp

```diff
@@ -475,7 +486,14 @@ DocumentPtr TextureBaker::bakeMaterialToDoc(DocumentPtr doc, const FileSearchPat
 
     DefaultColorManagementSystemPtr cms = DefaultColorManagementSystem::create(genContext.getShaderGenerator().getTarget());
     cms->loadLibrary(doc);
-    genContext.registerSourceCodeSearchPath(searchPath);
+    if (!_codeSearchPath.isEmpty())
+    {
+        genContext.registerSourceCodeSearchPath(_codeSearchPath);
+    }
+    for (const FilePath& path : searchPath)
+    {
+        genContext.registerSourceCodeSearchPath(path / "libraries");
```

Comment:
This looks like a reference to the older pattern where the string "libraries" was manually added to each location in the code search path.  This should no longer be needed in the latest build, and I would recommend double-checking that this logic is working as intended.